db/migrate/20190411125709_change_workflow_steps_default.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeWorkflowStepsDefault < ActiveRecord::Migration
+  def change
+    reversible do |dir|
+      dir.up do
+        change_column_default(:workflows, :steps, [])
+        Workflow.select(:id).find_in_batches do |workflows|
+          workflows_update_scope = Workflow.where(id: workflows)
```

Comment:
I kind of expect this to work but was surprised to see it without a pluck on the .id attribute. 
```suggestion
          workflows_update_scope = Workflow.where(id: workflows.pluck(:id))
```

Reply:
The `workflows` block parameter that `find_in_batches` uses is an array of workflow objects, not an AR relation, so it doesn't have a pluck method.